pdp/static/js/hydro_stn_controls.js

```diff
@@ -87,7 +87,7 @@ function addToSidebar(idx, dataArray) {
     });
 
     link = document.createElement('a');
-    link.href = pdp.data_root + "/hydro_stn_cmip5/" + dataArray[idx].FileName + '.ascii';
+    link.href = `${pdp.data_root}/hydro_stn_cmip5/${dataArray[idx].FileName}`;
```

Comment:
Why did we lose `.ascii` here?

Reply:
It's because CSV files from THREDDS don't support the `.ascii` extension. So, they're just downloaded as `filename.csv` instead of `filename.csv.csv` or `filename.csv.ascii`.